packages/fiori/src/TimelineItem.ts

```diff
@@ -169,6 +202,23 @@ class TimelineItem extends UI5Element implements ITimelineItem {
 		this.shadowRoot!.querySelector<Link>("[ui5-link]")?.focus();
 	}
 
+	static typeTextMappings(): Record<string, I18nText> {
+		return {
+			"Information": TIMELINE_ITEM_INFORMATION_STATUS_TEXT,
+			"Positive": TIMELINE_ITEM_POSITIVE_STATUS_TEXT,
+			"Negative": TIMELINE_ITEM_NEGATIVE_STATUS_TEXT,
+			"Critical": TIMELINE_ITEM_CRITICAL_STATUS_TEXT,
+		};
+	}
+
+	get timelineItemStatusText() {
+		return TimelineItem.i18nBundle.getText(TimelineItem.typeTextMappings()[this.status]);
+	}
+
+	get ariaDescriptionText() {
+		return this.accessibleDescription || this.timelineItemStatusText || undefined;
```

Comment:
The `accessibleDescription` text provided by the application will override the internal status text like this. We'll need to concatenate them if they're both present.

Reply:
as mentioned above, this will no longer be needed.